libraries/persistent_store/src/lib.rs

```diff
@@ -12,299 +12,297 @@
+//! | t\_{I + i} | = | t_I + \|h_{I + i}\| - \|h_I\| + i           |
+//! |            | ≤ | h\_I + V + (I + i) * Q + M - h\_I + i       |
+//! |            | = | (N - 1) × (Q - 1) - M + (I + i) × Q + M + i |
+//! |            | = | (N - 1) × (Q - 1) + (I + i) × Q + i         |
+//! |            | = | (I + i + N - 1) × Q + i - (N - 1)           |
+//! |            | ≤ | (I + i + N - 1) × Q                         |
 //!
-//! We also want to show that after `N - 1` compactions, the remaining capacity is
-//! available without compaction.
+//! We also want to show that after N - 1 compactions, the remaining capacity is available without
+//! compaction.
 //!
-//! ```text
-//! V - (t_{I + N - 1} - h_{I + N - 1}) >=    // The available words in the window.
-//!   C - (|t_{I + N - 1}| - |h_{I + N - 1}|) // The remaining capacity.
-//!   + 1                                     // Reserved for Clear.
-//! ```
+//! |   |                                               |                                   |
+//! | -:| --------------------------------------------- | --------------------------------- |
+//! |   | V - (t\_{I + N - 1} - h\_{I + N - 1})         | The available words in the window |
+//! | ≥ | C - (\|t\_{I + N - 1}\| - \|h\_{I + N - 1}\|) | The remaining capacity            |
+//! | + | 1                                             | Reserved for clear                |
 //!
-//! We can replace the definition of `C` and simplify:
+//! We can replace the definition of C and simplify:
 //!
-//! ```text
-//! V - (t_{I + N - 1} - h_{I + N - 1}) >= V - N - (|t_{I + N - 1}| - |h_{I + N - 1}|) + 1
-//! iff t_{I + N - 1} - h_{I + N - 1} <= |t_{I + N - 1}| - |h_{I + N - 1}| + N - 1
-//! ```
+//! |     |                                       |   |                                                       |
+//! | ---:| -------------------------------------:|:-:|:----------------------------------------------------- |
+//! |     | V - (t\_{I + N - 1} - h\_{I + N - 1}) | ≥ | V - N - (\|t\_{I + N - 1}\| - \|h\_{I + N - 1}\|) + 1 |
+//! | iff | t\_{I + N - 1} - h\_{I + N - 1}       | ≤ | \|t\_{I + N - 1}\| - \|h\_{I + N - 1}\| + N - 1       |
 //!
 //! We have the following properties:
 //!
-//! ```text
-//! t_{I + N - 1} = t_I + |h_{I + N - 1}| - |h_I| + N - 1
-//! |t_{I + N - 1}| - |h_{I + N - 1}| = |t_I| - |h_I| // Compaction preserves capacity.
-//! |h_{I + N - 1}| - |t_I| <= h_{I + N - 1} - t_I
-//! ```
+//!
+//! |                                         |   |                                              |        |
+//! | ---------------------------------------:|:-:|:-------------------------------------------- |:------ |
+//! | t\_{I + N - 1}                          | = | t\_I + \|h\_{I + N - 1}\| - \|h\_I\| + N - 1 |        |
+//! | \|t\_{I + N - 1}\| - \|h\_{I + N - 1}\| | = | \|t\_I\| - \|h\_I\|   | Compaction preserves capacity |
+//! | \|h\_{I + N - 1}\| - \|t\_I\|           | ≤ | h\_{I + N - 1} - t\_I |                               |
```

Comment:
This one is the first table which is not aligned. But people should not read those tables here, they should use rustdoc, so it shouldn't hurt.

Reply:
Yeah, this one was more obviously intended! LGTM